src/Compiler/CMakeLists.txt

```diff
@@ -24,6 +24,14 @@ add_custom_target(ExternalUtil DEPENDS ${FILE_GENERATE_DIR}/ExternalUtil.hpp)
 
 get_property(OMLibs GLOBAL PROPERTY ONNX_MLIR_LIBS)
 
+# Link LLVM libraries necessary to query which target architectures are configured.
+set(ExtraLLVMLibs)
+list(APPEND ExtraLLVMLibs "LLVMMC")
```

Comment:
MC can also be resolved to LLVMMC by using the same component function as below.

I would suggest adding an option to `add_onnx_mlir_library` to make the resolve for all of these happen automatically - this way it could be used in any of the libs and not just here.

Reply:
`LLVMSupport` is an example of another llvm library that is used multiple places and could be specified as just `Support` with the proposed change.

Reply:
I added a new option to the `add_onnx_mlir_library` function as suggested.